typescript-decorators/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "typescript-decorators",
+  "version": "1.0.0",
+  "description": "Example how to use typescript decorators with RequestHandler pattern to build Fastify application",
+  "main": "bin/app.js",
+  "scripts": {
+    "start": "node bin/app.js",
+    "try-me": "curl http://localhost:3000/get",
+    "build": "tsc",
+    "test": "mocha"
+  },
+  "keywords": [
+    "typescript",
+    "fastify"
+  ],
+  "author": "Andrey Chalkin <L2jLiga@gmail.com> (https://github.com/L2jLiga)",
+  "license": "MIT",
+  "dependencies": {
+    "fastify": "^2.3.0"
+  },
+  "devDependencies": {
+    "@types/node": "^12.0.0",
+    "mocha": "^6.1.4",
```

Comment:
we typically use tap for tests, would you mind changing that? tap 13 also supports typescript out of the box.

Reply:
Sure